### PR TITLE
Post questions/problems as issues on Github

### PR DESCRIPTION
**Pull request by pwilkins**
I added a friendly line in the README directing questions/problems to the site's Github site.  I also stream-lined the doc by including the README content into the index page, rather than duplicating the text.  Ready for review @pdpinch @carsongee @Ferdi

### CODE REVIEW

**Comment by pdpinch** [comment]
Thanks @pwilkins . Changes look good to me, but I wanted to look also at ReadTheDocs. 

Excuse my ignorance, but is there a doc build on read the docs I can review?

**Comment by pwilkins** [comment]
That's easy to set up.

**Comment by pwilkins** [comment]
This branch is visible on RTD here: http://mit-lti-flask-example.readthedocs.org/en/feature-pw-github-issues-for-support/

File: docs/index.rst [hunk 1]
```diff
@@ -1,26 +1,5 @@
...
-You will need both this app and the PyLTI library to create your own LTI
-provider.  Each sample contains only the code variations necessary to support
-its specific framework.  By creating an interface boundary between a sample
-provider and PyLTI, PyLTI manages the specific LTI features and each sample
-manages the specific requirements of its framework.  You can easily switch your
-custom provider from one framework to another.
-
-Please see the PyLTI README `https://github.com/mitodl/pylti
-<https://github.com/mitodl/pylti>`_ for a detailed description of the architecture.
+.. include:: ../README.rst
```
Comment (pdpinch) [inline docs/index.rst]: Oh, you already fixed the duplicate title!

Comment (pwilkins) [inline docs/index.rst]: Yes, it was a side effect of importing the README.  I elected to remove "MIT's".  If you want, I could add it back in.

Comment (pdpinch) [inline docs/index.rst]: Nope. Not a big fan of that kind of branding. 

I just rebuilt on readthedocs and it looks good. I'll merge once I get to the right window.

**Pull request merged**